### PR TITLE
Update webpack - Immersive Reader SDK - Dev Public 3rd Party pipeline error

### PR DESCRIPTION
**Pull request by cristobal-buenrostro**

### CODE REVIEW

File: js/yarn.lock [hunk 1]
```diff
@@ -3430,7 +3419,7 @@ strip-ansi@^5.0.0, strip-ansi@^5.1.0, strip-ansi@^5.2.0:
   resolved "https://registry.yarnpkg.com/strip-ansi/-/strip-ansi-5.2.0.tgz#8c9a536feb6afc962bdfa5b104a5091c1ad9c0ae"
   integrity sha512-DuRs1gKbBqsMKIZlrffwlug8MHkcnpjs5VPmL1PAh+mA30U0DTotfDZ0d2UUsXpPmPmMMJ6W773MaA3J+lbiWA==
   dependencies:
-    ansi-regex "^5.0.1"
+    ansi-regex "^4.1.0"
```
Comment (rwallerms) [inline js/yarn.lock:3422]: downgrade?

File: js/yarn.lock [hunk 2]
```diff
@@ -3615,7 +3604,7 @@ ts-loader@^5.3.3:
   dependencies:
     chalk "^2.3.0"
     enhanced-resolve "^4.0.0"
-    loader-utils "^3.2.1"
+    loader-utils "^1.0.2"
```
Comment (rwallerms) [inline js/yarn.lock:3607]: downgrade?

Comment (cristobal-buenrostro) [reply js/yarn.lock:3607]: yes, basically this upgrade impacted, that old version is a dependency.

File: js/yarn.lock (hunk 1 above)
Comment (cristobal-buenrostro) [reply js/yarn.lock:3422]: yes was downgraded by webpack install, it seems older version is a dependency

**Review by rwallerms — APPROVED** [review]

**Review by sharad-sharma-msft — APPROVED** [review]

**Pull request merged**